composer.lock

```diff
@@ -3264,61 +3265,6 @@
             "time": "2015-08-27 11:27:33"
         },
         {
-            "name": "mathiasverraes/money",
```

Comment:
I used only commands such as `composer require ext-fileinfo:*` so the unrelated changes in `composer.lock` are probably result of it being out of sync with `composer.lock`.